### ISSUE TITLE
Width of letterListContainerStyle

### ISSUE THREAD

**Issue by StefanTanaskovicSoftLab** [Kieran-McIntyre/react-native-section-alphabet-list#67]
Is there a way to increase width of letter List container style because its hard to click on the specific letter?
![Screenshot_20230925_103555_timeviewapp](https://github.com/Kieran-McIntyre/react-native-section-alphabet-list/assets/115802183/c0ae5f10-b579-4a62-8cb0-1b8c3ead5bf7)

**Comment by Raiden-16F7** [comment]
you can use indexContainerStyle and set width
indexContainerStyle ={{ width:20 }}
![image](https://github.com/Kieran-McIntyre/react-native-section-alphabet-list/assets/59868973/08318b6d-bf39-42a4-9053-b676f4e63488)

**Comment by dokselblackbit** [comment]
Yes. You can add width to indexLetterContainerStyle, letterListContainerStyle and indexContainerStyle
 indexLetterContainerStyle is wrapper of each letter
 letterListContainerStyle is flatlist (letter's list)
 indexContainerStyle is wrapper of flatlist
